Automattic/wp-calypso
# Signup: Remove the domains step

#### Changes proposed in this Pull Request

* Our new signup flow doesn't have `domains` step. This PR will take the step away from the `onboarding` flow for testing.

#### Testing instructions



* Run the following command in browser console to activate the `onboarding` signup flow.
```
localStorage.setItem( 'ABTests', '{"improvedOnboarding_20181023":"onboarding"}' );
```
* Create a new website.
* You should not see the `domains` step in the flow and the site should have an auto-generated `.wordpress.com` domain name.
